functions/package.json

```diff
@@ -5,6 +5,8 @@
     "@google-cloud/storage": "^0.8.0",
     "firebase-admin": "^4.1.3",
     "firebase-functions": "^0.5.2",
-    "jsonwebtoken": "^7.3.0"
+    "jsonwebtoken": "^7.3.0",
+    "request": "^2.81.0",
+    "typescript": "^2.2.1"
```

Comment:
I thought we build the `functions/*.ts` file before deploying and Firebase shouldn't care about it? If so then we can just remove it from the package file here.